### PR TITLE
TAU package: Include recent change for Ubuntu

### PR DESCRIPTION
**Pull request by jordialcaraz**
Select option -disable-no-pie-on-ubuntu for some Ubuntu systems

https://github.com/UO-OACISS/tau2/commit/823971df015f2d28bf565b267c758f2deefa0cc9

### CODE REVIEW

**Review by wspear — DISMISSED** [review]
This worked for me!

File: var/spack/repos/builtin/packages/tau/package.py [hunk 1]
```diff
@@ -106,6 +106,8 @@ class Tau(Package):
     )
     variant("dyninst", default=False, description="Activates dyninst support")
 
+    variant("disable-no-pie", default=False, description="Do not add -no-pie while linking.")
```
Comment (tldahlgren) [inline var/spack/repos/builtin/packages/tau/package.py]: Curious.  Is it worth noting in the description that the option relates to ubuntu?

**Review by wspear — DISMISSED** [review]

**Review by tldahlgren — DISMISSED** [review]
LGTM. Thanks.

File: var/spack/repos/builtin/packages/tau/package.py (hunk 1 above)
Comment (jordialcaraz) [reply var/spack/repos/builtin/packages/tau/package.py]: Added that this is for Ubuntu, thanks.

**Comment by jordialcaraz** [comment]
@spackbot fix style

**Review by tldahlgren — APPROVED** [review]

**Pull request merged**